changes/modules.go

```diff
@@ -0,0 +1,34 @@
+package changes
+
+import (
+	"golang.org/x/mod/modfile"
```

Comment:
Something worth considering, but the `go mod` command can return JSON structures back that describe the go mod file contents. We could avoid pulling in a dependency library if we structure ourselves around that.

Reply:
If the tool ends up being its own submodule, its dependencies will be isolated from the rest of the SDK. In that case, it seems beneficial to avoid running and parsing the output of `go` commands by pulling in some of the Go packages for working with modules.